src/screens/SearchBeers/SearchBeers.jsx

```diff
@@ -93,6 +147,10 @@ class SearchBeersComponent extends PureComponent {
       colorOfBeer: '',
       brewedBefore: '',
       food: '',
+      beerName: [],
```

Comment:
beerName should not be an array and it is not - it's a string

Reply:
Done!